.github/workflows/mynmr_tools_ci.yml

```diff
@@ -73,7 +73,37 @@ jobs:
           cd clients/java
           mvn compile
           mvn test
-      
+
+  # JavaScript Client (latest Node version)
+  javascript-latest-test:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Setup node
+        uses: actions/setup-node@v1
+        with:
+          node-version: 14.x
+      - name: Javascript client test
+        run: |
+          cd clients/js
+          npm ci
+          npm run test
+
+  # JavaScript Client (lower npm version)
```

Comment:
Preserve the rest of the comment in the Travis config?

```
    # Note: even older Node.js versions might be allowed, but the
    # test tooling requires more recent versions.
```

Reply:
I didn't include this comment because it may be Travis specific(?)

Reply:
The comment is not Travis-specific and it would be useful to include here.